recorder/src/main/resources/reference.conf

```diff
@@ -5,8 +5,8 @@ data.cluster = {
    resolveSecrets = false
 }
 
-data-source.index.used.name=used-concepts
+data-source.index.used.name=concepts-used
```

Comment:
Going to try consolidating a single source of truth for index names to avoid these mistakes from happening again